src/typed-method-types/workflows/triggers/base_response.ts

```diff
@@ -22,9 +22,16 @@ export type BaseTriggerResponse<
       /** @default "workflow" */
       type: string;
       /** @description the workflow's input parameters */
-      input_parameters: WorkflowDefinition["input_parameters"];
+      input_parameters: {
```

Comment:
I wonder if we should type both `input_parameters` and `output_parameters` as the same type?

Reply:
Honestly I am not sure. The output_parameters never had items for my testing

Reply:
Let's not worry about it too much for now, I can always investigate more deeply in webapp code if we find a need for it.